### PR TITLE
[Ide] Remove deprecated FeedbackService

### PR DESCRIPTION
**Pull request by sevoku**
As discussed in #5791, this PR removes the whole `FeedbackService`, which has not been used for years now. 

> Fixes VSTS [#668256](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/668256)

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Extensions/ServiceUrlExtensionNode.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@
 
 namespace MonoDevelop.Ide.Extensions
 {
+	[Obsolete]
 	public class ServiceUrlExtensionNode : ExtensionNode
```
Comment (sevoku) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Extensions/ServiceUrlExtensionNode.cs]: Only this class is public, adding `Obsolete` attribute to not hard break the API.

Comment (slluis) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Extensions/ServiceUrlExtensionNode.cs]: Let's remove it too. It should have never been public.

**Comment by slluis** [comment]
@monojenkins backport release-7.6

**Comment by Therzok** [comment]
Rebased on top of current master to fix the build.

**Pull request merged**